PresetParser/BuildingBlockProvider.cs

```diff
@@ -66,7 +66,22 @@ private bool ParseBuildingBlockerForAnno1800(XmlDocument ifoDocument, IBuildingI
                 {
                     xf = 3; zf = 3;
                 }
-
+                if (building.Identifier == "Harbor_arctic_01 (Depot)")
+                {
+                    xf = 11; zf = 4;
+                }
+                if (building.Identifier == "River_colony02_01 (Clay Harvester)")
+                {
+                    xf = 9; zf = 5;
+                }
+                if (building.Identifier == "River_colony02_02 (Paper Mill)")
+                {
+                    xf = 9; zf = 5;
+                }
+                if (building.Identifier == "River_colony02_03 (Water Pump)")
+                {
+                    xf = 9; zf = 5;
+                }
```

Comment:
What is this code doing? Might be worth adding a comment to explain it.

Reply:
Theres is a comment already, but it might be worth being a bit more explicit, especially with the canal stuff.

Reply:
The buildings there has a 5x1 measurement in the raw data, that is wrong size they are minimal 9x5
so i think the comment there is oke, as i fix the wrong size of a building

Reply:
The whole if section could be replaced by a single switch statement

Reply:
?

Reply:
Never mind i know what you mean now :D